src/sentry/integrations/pagerduty/utils.py

```diff
@@ -62,7 +62,7 @@ def send_incident_alert_notification(
     metric_value: int,
     new_status: IncidentStatus,
     notification_uuid: str | None = None,
-) -> None:
+) -> bool:
```

Comment:
does this not cause a typing error since a `bool` isn't always returned? same with `/opgenie/`

Reply:
for some reason some raise and others don't